components/climate/thermostat.rst

```diff
@@ -315,6 +315,93 @@ Set Point Options/Behavior
 **If configured, at least one of** ``default_target_temperature_low`` **and** ``default_target_temperature_high``
 **must be specified in the away mode configuration.**
 
+Presets
+*******
+
+Presets allow you to preconfigure different modes for your thermostat. These will show up in the Home Assistant
```

Comment:
I think I would avoid the word "modes" to describe what presets are. I might word it this way:

_"Presets allow you to define a combination of set points, climate, fan, and swing modes that can be recalled from the front end (Home Assistant) collectively for quick, easy access. This can simplify the user experience and automations, among other things."_